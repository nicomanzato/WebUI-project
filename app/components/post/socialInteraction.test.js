import React from 'react';
import SocialInteraction from './socialInteraction';
import { shallow, mount, ReactWrapper } from 'enzyme';

describe('social interaction', () => {

  it('should render correctly', () => {
    const wrapper = shallow(
      <SocialInteraction favoriteCount={10} retweetCount={15} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
