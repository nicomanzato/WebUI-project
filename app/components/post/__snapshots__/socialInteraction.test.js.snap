// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`social interaction should render correctly 1`] = `
<Component
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-between",
      "paddingLeft": 35,
      "paddingRight": 35,
      "paddingTop": 15,
    }
  }
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="grey"
      name="ios-heart-outline"
      size={32}
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "grey",
          "fontSize": 15,
          "margin": 5,
        }
      }
    >
      10
    </Text>
  </Component>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="grey"
      name="ios-chatboxes-outline"
      size={32}
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "grey",
          "fontSize": 15,
          "margin": 5,
        }
      }
    >
      0
    </Text>
  </Component>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="grey"
      name="ios-share-alt"
      size={32}
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "grey",
          "fontSize": 15,
          "margin": 5,
        }
      }
    >
      15
    </Text>
  </Component>
</Component>
`;
